README.md

```diff
@@ -51,6 +51,8 @@ import React from "react";
 import ReactDOM from "react-dom";
 import { ContextMenu, MenuItem, ContextMenuTrigger } from "react-contextmenu";
 
+import 'react-contextmenu/public/styles.5bb557.css';
```

Comment:
You should actually add the css styling your own css files
https://github.com/vkbansal/react-contextmenu/blob/master/docs/usage.md#styling